server/lib/nl/constants.py

```diff
@@ -166,11 +166,14 @@
                 "best",  # leaving here for backwards-compatibility
                 "highest",
                 "high",
-                "smallest",
+                "largest",
```

Comment:
Are there any tests for the classifier? Worth start to adding them?

Reply:
Tests are a good idea. Should I add to this PR or should I submit tests in a followup PR?

Reply:
Can add in this PR? There are some related tests can follow in https://github.com/datacommonsorg/website/tree/master/server/tests/lib/nl

Reply:
Great, I'll convert to draft for now, and re-request review once tests are added.

Reply:
Should be easy to add tests for heuristic classifications as it does not need mock or other setup?

Reply:
Added tests!

Reply:
very nice!